docs/pages/workflow/configuration.md

```diff
@@ -6,9 +6,11 @@ import PossibleRedirectNotification from '~/components/plugins/PossibleRedirectN
 
 <PossibleRedirectNotification newUrl="/versions/latest/config/app/" />
 
-**app.json** is your go-to place for configuring parts of your app that don't belong in code. It is located at the root of your project next to your **package.json**. It looks something like this:
+The Expo config (**app.json**, **app.config.js**) is used for configuring how a project loads in [Expo Go](https://expo.dev/expo-go), [Expo Prebuild](/workflow/prebuild.md) generation, and the OTA update manifest (think of this like an `index.html` for React Native apps).
```

Comment:
```suggestion
The Expo config (**app.json**, **app.config.js**, **app.config.ts**) is used for configuring how a project loads in [Expo Go](https://expo.dev/expo-go), [Expo Prebuild](/workflow/prebuild) generation, and the OTA update manifest (think of this like an `index.html` for React Native apps).
```

Can we add `app.config.ts` to the list or mark somehow that it is also a valid config file?

Reply:
It's an ok add but in general I think it was partially a mistake to add cuz it enforces certain language features making it not proper TypeScript support.